core/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -683,17 +684,24 @@ class QualificationAppInfo(
         1
       }
 
+      // This is the wallclock time for the SQL Dataframe tasks that we will use for the
+      // estimation of SQL spent on GPU. We exclude the job overhead time from the app duration
+      // to get the wallclock time for the SQL Dataframe tasks.
       val wallClockSqlDFToUse = QualificationAppInfo.wallClockSqlDataFrameToUse(
-        sparkSQLDFWallClockDuration, appDuration)
+        sparkSQLDFWallClockDuration, appDuration - jobOverheadTime)
```

Comment:
so this is technically still not correct because its ignoring nonSQLDataframeTaskDuration. But really subtracting either of those from a duration that isn't real is also wrong.  I think we need to do some ratio's and apply to the app  duration, need to think about it some more and will get back to you